mitosheet/src/types.tsx

```diff
@@ -206,17 +206,29 @@ export type GraphCreationParams = {
     y_axis_column_ids: ColumnID[]
     color: ColumnID | undefined
 }
-export type GraphStylingParams = undefined
-export type GraphRenderingParams = {
-    width?: number
-    height?: number
+export type GraphStylingParams = {
+    title: {
+        title: string | undefined,
```

Comment:
Comment about using the default when there is an undefined title, for clarity if you please